lib/reducers/create-otp-reducer.js

```diff
@@ -142,9 +143,21 @@ export function getInitialState(userDefinedConfig) {
   if (currentQuery.routingType === 'ITINERARY') {
     queryModes = ensureSingleAccessMode(queryModes)
   }
-  if (config.modes && config.modes.combinations) {
+  if (config.modes?.combinations) {
     // Inject combinations from config for batch routing.
-    currentQuery.combinations = clone(config.modes.combinations)
+    // Filter options based on defaults if modeOptions is specified in config
+    let enabledCombinations
```

Comment:
Might we be able to avoid a let here if we do the default modes above this line and then use a `||`? Usually I'm against such shorthands but there's really only 2 cases here